torch/_torch_docs.py

```diff
@@ -8390,6 +8413,49 @@ def merge_dicts(*dicts):
 
 """.format(**factory_common_args))
 
+add_docstr(torch.kaiser_window,
+           """
+kaiser_window(window_length, beta, periodic=True, dtype=None, \
```

Comment:
Mentioned this earlier, but the third parameter would be `sym=True` to be consistent with NumPy. The parameters from TensorOptions are also kwarg-only, right? So it'd be `sym=True, *, ...`